res/qml/Settings/SoundHardware.qml

```diff
@@ -0,0 +1,66 @@
+import QtQuick
+import Mixxx 1.0 as Mixxx
+
+Mixxx.SettingGroup {
+    id: root
+
+    property int selectedIndex: 0
+    property list<string> tabs: ["engine", "delays", "stats"]
```

Comment:
`SettingGroup` is a `QQuickItem`, how should this tabs thing along with the `selectedIndex` work?